### ISSUE TITLE
 TypeError [ERR_INVALID_URL]: Invalid URL: /8b43027f47b20503057dfbbaa9401fef.eot?#iefix&v=4.7.0

### ISSUE THREAD

**Issue by Slapbox** [webpack-contrib/css-minimizer-webpack-plugin#91]
Issue occurs with `css-minimizer-webpack-plugin@3.0.0` and `webpack@5.37.0`.

Same issue also occurs with `optimize-css-assets-webpack-plugin@6.x`, but not `5.x`. But perhaps that's expected since that plugin says to use `css-minimizer-webpack-plugin` instead when using `webpack@5.x`.

```
 ERROR in 945.css
[2] 945.css from Css Minimizer
[2] TypeError [ERR_INVALID_URL]: Invalid URL: /8b43027f47b20503057dfbbaa9401fef.eot?#iefix&v=4.7.0
[2]     at /home/me/Documents/MyProject/945.css:10:3
[2]     at onParseError (internal/url.js:257:9)
[2]     at new URL (internal/url.js:333:5)
[2]     at /home/me/Documents/MyProject/node_modules/css-minimizer-webpack-plugin/node_modules/postcss-svgo/dist/index.js:32:17
[2]     at walk (/home/me/Documents/MyProject/node_modules/postcss-value-parser/lib/walk.js:7:16)
[2]     at ValueParser.walk (/home/me/Documents/MyProject/node_modules/postcss-value-parser/lib/index.js:18:3)
[2]     at minify (/home/me/Documents/MyProject/node_modules/css-minimizer-webpack-plugin/node_modules/postcss-svgo/dist/index.js:22:23)
[2]     at /home/me/Documents/MyProject/node_modules/css-minimizer-webpack-plugin/node_modules/postcss-svgo/dist/index.js:116:9
[2]     at /home/me/Documents/MyProject/node_modules/css-minimizer-webpack-plugin/node_modules/postcss/lib/container.js:91:18
[2]     at /home/me/Documents/MyProject/node_modules/css-minimizer-webpack-plugin/node_modules/postcss/lib/container.js:74:18
[2]     at AtRule.each (/home/me/Documents/MyProject/node_modules/css-minimizer-webpack-plugin/node_modules/postcss/lib/container.js:60:16)
```

**Comment by alexander-akait** [comment]
Bug in `postcss-svgo`, please open an issue in `postcss-svgo`

**Comment by skytt** [comment]
There is someone fixed this problem with a PR on https://github.com/cssnano/cssnano/pull/1050

**Comment by Slapbox** [comment]
Good find @skytt. That was merged 3 days ago, and `CSSNano@5.0.5` was just released a few minutes ago, although the changes aren't yet outlined on the Releases page.
